src/dot/mod.rs

```diff
@@ -0,0 +1,99 @@
+use verkle_trie::proof::UpdateHint;
+use ark_serialize::{CanonicalSerialize};
+// use verkle_trie::EdwardsProjective;
+use std::borrow::Cow;
+
+use std::io::Write;
+
+type Nd = usize;
```

Comment:
don't abbreviate short words, just use `type Node = usize;` `type Edge = (usize, usize, u8);` and then you can use `&Edge`